src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead, style elements using classes. Consider adding a class to the `.stars` container and styling that class instead.